setup.cfg

```diff
@@ -1,6 +1,6 @@
 [flake8]
 ignore =
-    E501,F403,F405,E226,E265,E731,E402,E266,
+    E501,F403,F405,E226,E265,E731,E402,E266,F999,
```

Comment:
Maybe because you have a different global setup? I get hundreds of warnings without F999